src/main/java/org/websync/supportedframeworks/jdi/ComplexElementsInitialization.java

```diff
@@ -0,0 +1,154 @@
+package org.websync.supportedframeworks.jdi;
+
+
+import com.epam.jdi.light.elements.complex.*;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.complex.dropdown.DropdownExpand;
+import com.epam.jdi.light.elements.complex.dropdown.DropdownSelect;
+import com.epam.jdi.light.elements.complex.table.*;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.*;
+import com.epam.jdi.light.ui.html.elements.complex.DataListOptions;
+import com.epam.jdi.light.ui.html.elements.complex.MultiSelector;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import com.epam.jdi.light.ui.html.elements.complex.Tabs;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class ComplexElementsInitialization extends WebPage {
+    @Css(".testCss")
+    public DataListOptions initializedDataListOptions;
```

Comment:
Move to page object for issue:
https://github.com/websyncio/websync-idea/issues/46